src/main/java/blackjack/domain/card/CardDeck.java

```diff
@@ -44,4 +42,8 @@ private void validateCapacity(int count) {
             throw new IllegalArgumentException(OVER_CAPACITY_EXCEPTION_MESSAGE);
         }
     }
+
+    public LinkedList<Card> getCardDeck() {
```

Comment:
```suggestion
    public List<Card> getCardDeck() {
```

그리고 테스트 때문에 이 메서드를 추가한 것 같은데 getter 대신 size 메서드로 바꿔보시기 바랍니다.
getter는 캡슐화를 깨뜨릴 수 있습니다.

Reply:
getter 사용 지양..!
바로 적용했습니다!